zorg/buildbot/builders/LLDBBuilder.py

```diff
@@ -50,9 +53,18 @@ def getLLDBCMakeBuildFactory(
     lit_args = '-v'
 
     if jobs:
-        build_cmd.append(WithProperties("-j%s" % jobs))
-        install_cmd.append(WithProperties("-j%s" % jobs))
-        test_cmd.append(WithProperties("-j%s" % jobs))
+        jobs_option = ["-j{}".format(jobs)]
+    else:
+        # Use the worker property if option was not explicitly specified.
```

Comment:
Just some questions to help me understand this part.

Is each builder free to use the worker property `jobs` in the way that best fits it?
Is it ok for this function's jobs to be greater than worker's `jobs`?

Reply:
Good point. If you set jobs > number of cores, the best case is ninja uses the number of cores anyway, worst case you're creating some extra processes that have to wait around. Certainly there's no good reason to do this.

Given that, I think it would be better to remove the `jobs=` parameter and only use the worker property (if present).

Reply:
I've done this and updated the title/commit message to explain that. Much simpler this way.